confmap/provider/httpsprovider/README.md

```diff
@@ -0,0 +1,15 @@
+What is this new component httpsprovider?
```

Comment:
Could you format  the section titles as Markdown titles? e.g. here

```suggestion
### What is this new component httpsprovider?
```

(and also below)

Reply:
Done!